client/my-sites/domains/domain-management/edit/domain-types/mapped-domain-type.jsx

```diff
@@ -70,16 +77,20 @@ class MappedDomainType extends React.Component {
 				}
 			);
 		} else {
+			setupInstructionsMessage = translate(
+				'Follow these instructions to set up your domain mapping:'
+			);
 			primaryMessage = translate(
-				'Your domain mapping has not been set up. You need to update your name servers at the company where you purchased the domain to:',
+				'In order to connect your domain to WordPress.com, please log into your account at your domain registrar and update the name servers of your domain to use the following values, as detailed in {{link}}these instructions{{/link}}:',
```

Comment:
If we go with the above, we can drop `In order to connect your domain to WordPress.com,
` here.

How about we make the `update the name servers` part `strong`?

<img width="665" alt="Screenshot 2021-03-10 at 14 45 21" src="https://user-images.githubusercontent.com/3392497/110646883-41e94a80-81af-11eb-88f3-96c2878441fa.png">

Just to help skim the options/instructions.